source/Xunit.Gherkin.Quick.ProjectConsumer/Addition/AddTwoNumbers.cs

```diff
@@ -11,19 +11,19 @@ public sealed class AddTwoNumbers : Feature
         public AddTwoNumbers(ITestOutputHelper output) : base(output)
         {}
 
-        [Given(@"I chose (\d+) as first number")]
+        [GivenAttribute(@"I chose (\d+) as first number")]
```

Comment:
Can you remove Attribute portion of it? let's make it a good example for others.
Attributes don't require to be fully specified when applied.
Same below.